### PR TITLE
Use CDN version of FiraCode for the engine profiler

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: editor/resources/engine-profiler/remotery/vis/Styles/Remotery.css [hunk 1]
```diff
@@ -32,7 +32,7 @@ body
 @font-face
 {
     font-family: "LocalFiraCode";
-    src:url("Fonts/FiraCode/FiraCode-Regular.ttf");
+    src:url("https://cdnjs.cloudflare.com/ajax/libs/firacode/5.2.0/ttf/FiraCode-Regular.ttf") format('truetype');
```
Comment (JCash) [inline editor/resources/engine-profiler/remotery/vis/Styles/Remotery.css:35]: I couldn't patch this path to work with the engine profiler prefix, so I went with a global path.

File: editor/src/clj/editor/engine_profiler.clj [hunk 2]
```diff
@@ -48,6 +48,7 @@
                     "json" "application/json"
                     "css" "text/css"
                     "ttf" "font/ttf"
+                    "glsl" "text/plain"
```
Comment (JCash) [inline editor/src/clj/editor/engine_profiler.clj:51]: The recommended mime type according to glTF

File: engine/dlib/src/remotery/README.md [hunk 3]
```diff
@@ -16,12 +16,25 @@ Copy the HTML files to the editor:
 
     cp -v -r ~/notwork/Remotery/vis/ editor/resources/engine-profiler/remotery/vis
 
+Remove the font(s)
+
+    rm -rf editor/resources/engine-profiler/remotery/vis/Styles/Fonts
```
Comment (JCash) [inline engine/dlib/src/remotery/README.md:21]: If we're not using them, then let's not ship the files.

**Review by britzl — APPROVED** [review]
Looks good!

**Pull request merged**